app/routes/workshops/index.tsx

```diff
@@ -27,26 +34,134 @@ export function meta() {
+    description = 'Description TBA',
+    tech,
+  },
+}: MdxListItem) {
+  return (
+    <Link
+      to={slug}
+      className="focus-ring block flex flex-col p-16 pr-24 w-full h-full bg-gray-100 dark:bg-gray-800 rounded-lg"
+    >
+      <div className="flex-none h-36">
+        {/* TODO: how to determine if it's open or not? */}
+        {Math.random() * 10 < 5 ? (
+          <div className="inline-flex items-baseline">
+            <div className="block flex-none w-3 h-3 bg-green-600 rounded-full" />
+            <H6 as="p" className="pl-4">
+              Open for registration
+            </H6>
+          </div>
+        ) : null}
+      </div>
+      <div className="flex-none">
+        <div className="inline-block mb-4 px-8 py-4 text-black dark:text-white text-lg dark:bg-gray-600 bg-white rounded-full">
+          {tech}
+        </div>
+      </div>
+      <H3 className="flex-none mb-3">{title}</H3>
+      {/*TODO: line-clamp has a browser support of 90%, is that enough?*/}
+      <div className="flex-auto mb-10">
+        <Paragraph className="line-clamp-3">
+          {/*
+            We do use css line-clamp, this is for the 10% of the browsers that
+            don't support that. Don't focus too much at perfection. It's important
+            that the truncated string remains longer than the line-clamp, so that
+            line-clamp precedes for the 90% supporting that.
+          */}
+          {truncate(description, 120)}
+        </Paragraph>
+      </div>
+      {/*
+        TODO: design shows different date format, but that wraps really quickly.
+         I think we can move the time with timezone to the details page?
+      */}
+      <H6 className="flex flex-wrap">
+        {date ? formatDate(new Date(date), 'PPP') : 'To be announced'}
+      </H6>
+    </Link>
+  )
+}
+
+const ALL_TECHS = ['javascript', 'react', 'testing']
+
 function WorkshopsHome() {
   const data = useRouteData<LoaderData>()
+  // TODO: save state in url, like the filters on the blog?
+  const [selectedTech, setSelectedTech] = useState<string>('')
+  const workshops = selectedTech
+    ? data.workshops.filter(x => x.frontmatter.tech === selectedTech)
+    : data.workshops
+
   return (
     <div>
-      <header>
-        <h2>Kent C. Dodds</h2>
-      </header>
-      <main>
-        {data.workshops.map(workshop => (
-          <p key={workshop.slug}>
-            <Link to={`/workshops/${workshop.slug}`}>
-              {workshop.frontmatter.title ?? 'Untitled Workshop'}
-            </Link>
-            <br />
-            <small>
-              {workshop.frontmatter.description ?? 'Description TBA'}
-            </small>
-          </p>
+      <Grid className="grid-rows-max-content mb-36 mt-16">
+        <div className="col-span-full lg:col-span-6 lg:col-start-7 lg:row-span-2">
+          <img
+            className="object-cover"
+            src={images.teslaX.src}
+            alt={images.teslaX.alt}
+          />
```

Comment:
Image positioning should be fixed when the new images get uploaded. If not, we'll catch it in the second round.